src/commonMain/kotlin/com/github/quillraven/fleks/family.kt

```diff
@@ -108,7 +110,7 @@ data class Family(
             if (isDirty && !isIterating) {
                 // no iteration in process -> update entities if necessary
                 isDirty = false
-                entityBits.toEntityBag(field)
+                privateEntities.toEntityBag(field)
```

Comment:
I'd inline the code from below since it is only used once and is only two lines of code